contracts/linear/src/events.rs

```diff
@@ -117,10 +117,10 @@ pub enum Event<'a> {
         account_id: &'a AccountId,
         weight: u16,
     },
-    ValidatorUpdatedWeight {
-        account_id: &'a AccountId,
-        old_weight: u16,
-        new_weight: u16,
+    ValidatorUpdatedWeights {
```

Comment:
I think `ValidatorsUpdatedWeights` is better, indicating that multiple validators' weights are changed.